modal/modal.css

```diff
@@ -0,0 +1,76 @@
+@import "../resetcss.css";
+
+.skin{
+    padding: 0.5rem 0.5rem 0 0.1rem;
+    
+}
+.card-skin{
+    margin: 1rem;
+    width: 35rem;
+}
+
+a{
+  font-size: 2rem;
+  text-decoration: none;
+  color: black;
+}
+
+  article a{
+      display: block;
+      margin-top: 1rem;
+      padding-left: 2rem;
+      
+  
+  }
+  article{
+      display: flex;
+      flex-direction: column;
+      gap: 1rem;
+      /* margin : 2rem 0 8rem 0; */
+  }
+  
+
+    /* ---------------------------------------------------------------------------------------------------- */
+
+    .open-button{
+        outline: none;
+        padding: 1rem 2rem;
+        color: var(--primary-color);
+        border: var(--primary-color);
+        margin: 2rem;
+      }
+      .modal-box{
+        border: 1px solid black;
+        position: absolute;
+        left: 40%;
+        top: 60%;
+       border-radius: 1rem;
+        
+      }
+      .modal-button{
+        display: flex;
+        justify-content: right;
+        margin: 1rem;
+      }
+      .m_button{
+        outline: none;
+        padding: 0.5rem 1rem;
+        color: var(--primary-color);
+        border: var(--secondary-color)
+      }
+      .m_button:hover{
+        background-color: var(--primary-color);
+        color: var(--background-color);
+      }
+      /* ----------------------------------------------------------------------------------- */
+
+      #modal-content{
+        border: 1px solid var(--primary-color);
+        width: 40rem;
+        padding: 1rem;
+        visibility: hidden;
+      }
+      .modal:active{
+        background-color: var(--primary-color);
+        color: var(--background-color);
+      }
```

Comment:
The only suggestion is to look at code formatting. Could use consistent spacings and tabs 👍🏽

Reply:
thanx bharati, i will sure look into this